src/lib/MUIDropDownTimePicker.tsx

```diff
@@ -1,12 +1,30 @@
-import { FormControl, FormControlProps, InputLabel, InputLabelProps, MenuItem, MenuItemProps, Select, FormHelperText } from '@material-ui/core';
+import { FormControl, FormControlProps, InputLabel, InputLabelProps, MenuItem, MenuItemProps, Select, FormHelperText, SelectProps, FormHelperTextProps } from '@material-ui/core';
 import { FormikValues } from 'formik';
 import { get, isString, map } from 'lodash';
 import React, { FC } from 'react';
 import moment from 'moment'
-import { FormConfig } from 'react-forms';
-import { IMUIDropDownTimePickerProps, MUIDropDownTimePickerProps } from 'react-forms/dist/lib/ml-form-builder/lib/MUIDropDownTimePicker';
+import { FormConfig, IFieldProps } from 'react-forms';
+// import { IMUIDropDownTimePickerProps, MUIDropDownTimePickerProps } from 'react-forms/dist/lib/ml-form-builder/lib/MUIDropDownTimePicker';
+// import { getFieldError, MenuOptionObject } from 'react-forms/dist/lib/ml-form-builder/Utils';
 import { getFieldError, MenuOptionObject } from './Utils';
 
+export interface IMUIDropDownTimePickerProps extends SelectProps {
```

Comment:
Change MUIDropDownTimePickerProps to DropDownTimePickerFieldProps

Reply:
@sirajalam049 Done the requested changes.